src/client/Abfahrten/Components/FavList.tsx

```diff
@@ -78,11 +78,20 @@ const FavList = ({ staticContext }: Props) => {
           <MostUsed />
         </>
       ) : sortedFavs.length ? (
-        sortedFavs
+        <>
+          <FavEntryDisplay className="nonClickable" text="Favoriten" />
```

Comment:
Keine Statischen ClassNames.
```suggestion
            className={classes.nonClickable}
```